### ISSUE TITLE
The Viewport Cluster UI overlay is appearing on top of over viewports (like Script Canvas) when using picker

### ISSUE THREAD

**Issue by hultonha** [o3de/o3de#8193]
[Migrated from JIRA LYN-8190]
Provide a detailed description of the defect here and include any possible workarounds.
----
*Found in Build*: Daily Build exe for o3de jam

*Reproducibility*: 3/3

*Prerequisites*: Open up script canvas, and float it on top of the editor

*Steps to Reproduce*:
 # Open script canvas and overlay it on top of the editor
 # add some script canvas nodes
 # click on the crosshair picker
 # the viewport cluster UI (from the editor) is appearing on top. it confuses me.

*Expected Result*:

1. It to not appear.

*Actual Result*:

It is appearing on top of the script canvas viewport.

 

Video attached of the issue.

**Comment by hultonha** [comment]
Closing as duplicate of https://github.com/o3de/o3de/issues/2603
